### PR TITLE
feat: make public Flux::QueryResult::Table(T) private getter records

### PR DESCRIPTION
**Pull request by dukenguyenxyz**
- Make `private getter records` from `Flux::QueryResult::Table(T)` `public`
- Bump tasker to v2

### CODE REVIEW

File: src/flux/query_result/table.cr [hunk 1]
```diff
@@ -5,7 +5,7 @@ struct Flux::QueryResult::Table(T)
 
   getter columns : Array(Column)
 
-  private getter records = [] of T
+  getter records = [] of T
```
Comment (kimburgess) [inline src/flux/query_result/table.cr:8]: This is an internal data structure.

`Table(T)` is [`Indexable`](https://crystal-lang.org/api/0.36.1/Indexable.html) over the elements this holds.

**Review by kimburgess — CHANGES_REQUESTED** [review]
See comment - exposing this should not be required.

**Comment by kimburgess** [comment]
Bumping to the fresh Tasker release is worthwhile though! But would be nice to seperate PR's for these two things.

File: src/flux/query_result/table.cr (hunk 1 above)
Comment (dukenguyenxyz) [reply src/flux/query_result/table.cr:8]: @GabFitzgerald

**Pull request closed without merging**